test/performance/odp_bench_misc.c

```diff
@@ -1038,6 +1085,13 @@ int main(int argc, char *argv[])
 
 	ret = gbl_args->suite.retval;
 
+	if (common_options.is_export) {
+		if (bench_misc_export(gbl_args)) {
+			ODPH_ERR("Error: Export failed\n");
+			ret = -1;
```

Comment:
This way it's easy to detect failures in CI pipelines, which record performance results. If the test is otherwise passing it could take a while until one notices that not all results are recorded correctly.